pkgs/by-name/fa/factorio_achievement_enabler/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  ninja,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fae_linux";
```

Comment:
Usually pname would be equal to the package name (`factorio_achievement_enabler`). The commit message must reference the package name.